src/assets/icon/AnonymousIcon.js

```diff
@@ -1,23 +1,24 @@
 import * as React from 'react';
 import Svg, {ClipPath, Defs, G, Path, Rect} from 'react-native-svg';
+import {COLORS} from '../../utils/theme';
 
 const AnonymousIcon = (props) => (
   <Svg
-    width={32}
-    height={32}
+    width={props.size || 32}
+    height={props.size || 32}
     fill="none"
     xmlns="http://www.w3.org/2000/svg"
     viewBox="0 0 32 32"
     {...props}>
-    <Rect width={32} height={32} rx={16} fill={props.fill || '#333'} />
+    <Rect width={32} height={32} rx={16} fill={props.fill || COLORS.signed_primary} />
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The fill color of the `Rect` component has been updated to use the `signed_primary` color from the new `COLORS` object instead of a hardcoded value. This is a good practice as it allows for easier updates in the future if the color scheme changes.



<!-- This is an auto-generated comment by OSS CodeRabbit -->